### PR TITLE
orcid-sync: force allow_push on tokens in Labs INSPIR-44

### PR DESCRIPTION
**Pull request by puntonim**

### CODE REVIEW

File: inspirehep/modules/orcid/tasks.py [hunk 1]
```diff
@@ -84,23 +84,30 @@ def _link_user_and_token(user, name, orcid, token):
...
-            secret=None,
-            extra_data={
-                'orcid': orcid,
-                'full_name': name,
-                'allow_push': True,
-            }
-        ))
+    if tokens:
+        # Force the allow_push.
+        with db.session.begin_nested():
```
Comment (michamos) [inline inspirehep/modules/orcid/tasks.py]: why the nested?

File: tests/integration/orcid/test_orcid_import_legacy_tokens.py [hunk 2]
```diff
@@ -75,7 +75,12 @@ def push_to_redis(user_record):
 def assert_db_has_n_legacy_records(n, record):
```
Comment (michamos) [inline tests/integration/orcid/test_orcid_import_legacy_tokens.py]: maybe s/records/tokens/ ?

File: inspirehep/modules/orcid/tasks.py (hunk 1 above)
Comment (puntonim) [reply inspirehep/modules/orcid/tasks.py]: Discussed. Done.

**Comment by puntonim** [comment]
ci merge please

**Comment by david-caro** [comment]
ci merge please

**Comment by puntonim** [comment]
ci merge please

**Comment by michamos** [comment]
ci merge please

**Pull request merged**